processing/src/main/java/io/druid/query/groupby/GroupByQuery.java

```diff
@@ -308,6 +308,7 @@ public GroupByQuery withDimensionSpecs(final List<DimensionSpec> dimensionSpecs)
     private LimitSpec limitSpec = null;
     private List<OrderByColumnSpec> orderByColumnSpecs = Lists.newArrayList();
     private int limit = Integer.MAX_VALUE;
+    private QueryGranularity orderingGranularity;
```

Comment:
With #2041, we can do something like that in best effort try. Anyway it will not do any harm on result except it'll introduce some inefficiency making it.